_posts/randy/2022-07-03-all-old-movies-are-garbage.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "All Old Movies Are Garbage (In the End)"
+permalink: /@randy/old-movies-are-garbage
+image: "assets/img/old-movies-are-garbage.png"
+description: Mine 
```

Comment:
Give a 1-2 sentence summary of the article (shows on article list and when shared on social media)

Reply:
thrown over the wall too soon